pkg/env/env.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright © 2021 The MayaData Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package env
+
+import (
+	"os"
+	"strings"
+)
+
+var (
+	// NFSServerNamespace defines the namespace of NFS Server resources
+	NFSServerNamespace = "OPENEBS_IO_NFS_SERVER_NS"
```

Comment:
lets use `openebs/dynamic-nfs-provisioner`  package for nfs-server related config params

Reply:
Thought about it(easy for maintenance) but avoided it since we have to do the import again.

Reply:
Updating to use `openebs/dynamic-nfs-provisioner` is causing other issues since volume-event-exporter is using v0.21.3 version whereas `openebs/maya`(indirect dependency via `openebs/dynamic-nfs-provisioner`) is using old version v0.17.3. So I didn't update this like will take this change later in the future.

Reply:
ok